obd/elm327.py

```diff
@@ -442,14 +466,20 @@ def __read(self):
 
             # if nothing was recieved
             if not data:
-                logger.warning("Failed to read port")
+                logger.warning("Failed to read port: empty data.")
                 break
 
             buffer.extend(data)
 
             # end on chevron (ELM prompt character)
             if self.ELM_PROMPT in buffer:
                 break
+         
+        #Check errors received by ELM327 and write debug
+        for iError in self._ERROR_MESSAGES:
```

Comment:
minor nit-pick: in python, we typically prefer lower-case, underscore separated names. For this, simply `error` would suffice (especially if we used the dictionary approach I mentioned above, which iterates over keys).